src/ScheduledTask/ProcessNotificationsHandler.php

```diff
@@ -198,6 +205,18 @@ public function run(): void
                 if (!$notification->isSuccess()) {
                     $this->handleFailedNotification($notification, $order, $state);
                 }
+/* gesagt.getan. begin -----------------------------------------------------------------------------------------------*/
```

Comment:
Thanks for the indicators, but this can also be seen in the Git history, I suggest removing them to keep the code clean
```suggestion
```